### PR TITLE
Fix for confirmation button layout

### PR DESCRIPTION
**Pull request by samageloff**

### CODE REVIEW

**Comment by samageloff** [comment]
[(1) React{ions} - Handle changes for ButtonConfirmation layout](https://trello.com/c/g5MEhWsd/88-1-react-ions-handle-changes-for-buttonconfirmation-layout)

File: src/components/Button/ButtonConfirmation.js [hunk 1]
```diff
@@ -98,24 +106,19 @@ export class ButtonConfirmation extends Component {
 
     // https://developers.google.com/web/updates/2016/06/absolute-positioned-children
     // Once ^ is supported in Safari and Firefox we can remove this and allow flex box to do its thing
-    if (this.props.collapse) {
-      return {
-        left: `-${((this.state.overlayWidth - this.state.triggerWidth) / 2 + 2)}px`
-      }
-    }
```
Comment (thomaslogangraves) [inline src/components/Button/ButtonConfirmation.js]: So this is solved for firefox and safari now without the additional code needed?

File: src/components/Button/ButtonConfirmation.js [hunk 2]
```diff
@@ -98,24 +106,19 @@ export class ButtonConfirmation extends Component {
...
     if (this.props.position === 'right') {
-      return { right: `calc(${(this.state.triggerWidth / 2) - 5}px)`}
+      return { right: `calc(${(this.state.triggerWidth / 2) + 7.5}px)`}
     }
     if (this.props.position === 'left') {
-      return { left: `calc(0% + ${(this.state.triggerWidth / 2) - 5}px)` }
+      return { left: `calc(${(this.state.triggerWidth / 2) - 5}px)` }
     }
-    return this.state.wide ? { left: `calc(0% + 96px)` } :  { left: `calc(0% + 66px)` }
+    return this.state.wide ? { left: `96px` } :  { left: `66px` }
```
Comment (thomaslogangraves) [inline src/components/Button/ButtonConfirmation.js]: This seems to be slightly off when collapsed. I put a screen shot on the card showing it off. Needs to be reduced by one px when collapsed.

File: src/components/Button/ButtonConfirmation.js (hunk 1 above)
Comment (samageloff) [inline src/components/Button/ButtonConfirmation.js]: No, the code is still there. It's just below this ...

Comment (thomaslogangraves) [inline src/components/Button/ButtonConfirmation.js]: 👍

**Pull request merged**